corehq/apps/app_manager/templates/app_manager/partials/modules/module_view_case_type.html

```diff
@@ -5,10 +5,17 @@
 <div class="form-group" id="case_type_form_group">
   <label class="{% css_label_class %} control-label">
     {% trans "Case Type" %}
+    {% if request|toggle_enabled:'USH_CASE_CLAIM_UPDATES' %}
+    <span class="hq-help-template"
+          data-title="{% trans "Case Type" %}"
+          data-content="{% trans "The name for the type of case you are tracking (e.g. “pregnancy,” “farm,” “household,” etc.). All cases you register in this case list will be this type of case. Other case lists with the same case type will use the same set of case(s)." %}"
```

Comment:
Minor: this should use `trans_html_attr`, which will handle quotes properly in translated text so it doesn't break the HTML.